middlewares/upload.js

```diff
@@ -0,0 +1,16 @@
+const multer = require('multer');
+const path = require('path');
+const uploadDir = path.join(process.cwd(), 'tmp');
```

Comment:
W folderach tmp i /public/avatars dodaj sobie pusty plik .gitkeep wtedy folder będzie zapisywany na gh (teraz tmp nie masz), a avatars wyrzucił bym zdjęcie i nie trzymał go w repo. (wybraź sobie że będzie tych zdjęć 100 i teraz ktoś ma ściągnąć to repo, to zajmie mu to chwilę)